### PR TITLE
(website): convert website to ts

### PR DESCRIPTION
**Pull request by AnthonyQuy**
## Proposed changes
Convert website to ts https://github.com/webdriverio/webdriverio/issues/10117

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [x] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

File: website/community/Team.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ id: team
 title: Team
 ---
 
-import Card from '@site/src/theme/card.jsx';
+import Card from '@site/src/theme/card.tsx';
```
Comment (christian-bromann) [inline website/community/Team.md:6]: We should add the `Card` component to `website/src/theme/MDXComponents.js` and remove all imports.

**Review by christian-bromann — COMMENTED** [review]
One comment, otherwise looks good

**Comment by AnthonyQuy** [comment]
Updated as comment

File: website/i18n/es/docusaurus-plugin-content-docs-community/current/Team.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ id: team
 title: El equipo
 ---
 
-import Card from '@site/src/theme/card.jsx';
+import Card from '@site/src/theme/card.tsx';
```
Comment (christian-bromann) [inline website/i18n/es/docusaurus-plugin-content-docs-community/current/Team.md:6]:
```suggestion
```

**Review by christian-bromann — APPROVED** [review]
LGTM 👍 

Thanks a lot!

**Comment by christian-bromann** [comment]
Congrats @AnthonyQuy on your first WebdriverIO contribution 🎉

**Comment by AnthonyQuy** [comment]
Thanks @christian-bromann, being use wdio for awhile and really like the project!

**Comment by christian-bromann** [comment]
Looking forward to more contributions. Let me know if I can help along the process.

**Pull request merged**